### PR TITLE
Mask sensitive objects

### LINKED ISSUE #8902: accounts_password_set_max_life_existing check outputs password hashes in the report
#### Description of problem:
Tests "xccdf_org.ssgproject.content_rule_accounts_password_set_max_life_existing" and "xccdf_org.ssgproject.content_rule_accounts_password_set_min_life_existing" result in the password hashes being included in the test report - this doesn't seem like a wise move.

#### SCAP Security Guide Version:
01.60

#### Operating System Version:
RHEL8

#### Steps to Reproduce:

1. Run ```oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis --report cis_report.html /usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml```
2. Check the HTML report for CIS 5.5.1.1 and 5.5.1.2 the "Existing Passwords" sections
3. See password hashes in HTML

#### Actual Results:

Password hashes in HTML report

#### Expected Results:

No password hashes in HTML report

#### Additional Information/Debugging Steps:

NA

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

In some rules, sensitive information might be shown in reports, even when they are not necessary.
It was included the `mask` parameter to hidden sensitive information from reports.

#### Rationale:

- Fixes #8902

### CODE REVIEW

**Comment by marcusburghardt** [comment]
Ok. Let me check why it is failing in container and adjust the test scenario script.

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh [hunk 1]
```diff
@@ -7,5 +7,5 @@
 {{{ bash_instantiate_variables("var_accounts_maximum_age_login_defs") }}}
 
 {{% call iterate_over_command_output("i", "awk -v var=\"$var_accounts_maximum_age_login_defs\" -F: '$5 > var || $5 == \"\" {print $1}' /etc/shadow") -%}}
-passwd -x $var_accounts_maximum_age_login_defs $i
+chage -M $var_accounts_maximum_age_login_defs $i
```
Comment (yuumasato) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh:10]: Update `fixtext` in the rule too.

**Review by yuumasato — CHANGES_REQUESTED** [review]
Looks great, just one small thing necessary.

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh (hunk 1 above)
Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh:10]: Done

**Review by yuumasato — APPROVED** [review]

**Pull request merged**